src/pages/LoginPage.js

```diff
@@ -63,6 +64,13 @@ const LoginPage = ({ logo }) => {
     history.push("/forgot_password");
   };
 
+  const errorMessages = {
+    GENERAL: formatMessage("core.LoginPage.authError"),
+    HF_CONTRACT_INVALID: formatMessage("core.LoginPage.authErrorHealthFacilityContractInvalid"),
+  };
+
+  const getErrorMessage = (key) => errorMessages[key] || errorMessages.GENERAL;
```

Comment:
1. Object with keys could be extendable but it's not the case for modular imis in which the authentication errors might come from different modules. Assuming that custom auth rules are added for insuree, AI module, social protection and so on it'd require changes in core and therefore loading logic should be different. However, I think this would be too much of a change for this ticket. Please create ticket on the OP space for that. 
2. We can address it later on. 

 I'd still like to have proper fallback added in scope of this PR. It's not a big of a change and it can help us avoid issues with incorrect error messages.

Reply:
I am unable to see how it should look like without changing the backend, what fallback would you like to see?